content/blog/cosign-2.5.0.md

```diff
@@ -22,7 +22,9 @@ Full Documentation you will be able to find [here](https://goharbor.io/docs/2.5.
 
 ## Demo Setup 
 **Configure two Harbor instances with Cosign enabled(default in v2.5.0) per repo and configure replication**
-We will have two instances harbor1 and harbor2, project "cosign" and replication rule(push based) between harbor1-to-harbor2, using robo-account.
+We will have two instances harbor1 and harbor2, project "cosign" and replication rule(push based) between harbor1-to-harbor2, using robo-account.  
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/u8nxky-6Q9c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>  
```

Comment:
Maybe just add a quick note before the video that says something like, "Check out a video of this demo"?

Reply:
added, didn't want to overcrowd the blog